frontend/src/pages/EventDetailsPage/components/AttendanceTable/index.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { useEffect, useState } from 'react';
 import { parseISO, format } from 'date-fns';
 
+import AttendanceDeleteButton from '../buttons/AttendanceDeleteButton';
```

Comment:
You didn't change this did you? I'm guessing this was Prettier just doing its job? I see all the files related to delete attendance have their formatting changed, but not the actual content itself

Reply:
I didn't change it. I think vs code somehow reformatted those and not the content.